eliHenderson/assignments/twopointers/w1d2/lc1.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Problem Statement:
+ *  Given an array of integers nums and integer target, return
+ * indices of the two numbers such that they add up to target.
+ *  You may assume that each input would have exactly one solution, and you may not use the same
+ * element twice
+ *  You can return and return the answer.
+ *
+ * Questions:
+ *  Is it sorted
+ *  can i use js specific methods
+ *
+ * overview :
+ *  set 2 pointers
+ *  1 at the start
+ *  1 at the end of the array
+ *  set a while loop to go until start pointer = end - 1
+ *  if the sum of the two indexes based on the pointers is greater than the target
```

Comment:
Nice overview here - I think you might mean if the sum is less than here though :)